doc/getting_started.md

```diff
@@ -0,0 +1,155 @@
+Include all Torque Tiles Specifications | [Torque tile specifications](#torque-tiles-specifications) define the TorqueMap Metadata and tileset information.<br /><br />**Note:** All Torque tile fields in the specification are required.
+Customize your Animation | Style and customize your animations with [CartoCSS Properties for Torque Style Maps](/cartodb-platform/cartocss/properties-for-torque/)
+
+
+## Torque Library
+
+Torque lets you render bigdata&trade;, time series or categorical data interactively. This is useful when you need to go beyond a static map, and create visualizations from temporal datasets. You can customize how your animated data appears in order to map such things as human movement, Twitter activity, biodiversity data, and other large-scale datasets. 
+
+- Export Torque tiles from your CartoDB account, or use a local Postgres installation [https://github.com/CartoDB/torque-gen](https://github.com/CartoDB/torque-gen)
+
+- Install [https://github.com/CartoDB/torque/blob/master/dist/torque.js](https://github.com/CartoDB/torque/blob/master/dist/torque.js)
+
+
+## Advanced Torque.js Libraries
+
+If you are interested in using advanced interaction methods, it is a prerequisite to load the Torque.js library before using the advanced interaction methods.
+
+- Download the [latest release](https://github.com/CartoDB/torque/releases) of the Torque.js source code
+
+
+## Torque Tiles Specifications
+
+Torque tiles are JSON representations of multidimensional data, with geospatial coordinates, that utilizes client-side resolution for rendering data. This optimizes the transfer of data for your Torque maps. Torque tile specifications are defined by two document types, Metadata and Tiles. The Metadata document describes the shared information across the Torque tile dataset. For each tile requested, there is a Tile document returned, which describes the data for that tile.
+
+### Metadata
+
+The TorqueMap Metadata document describes key tileset information, and includes the following fields:
+
+Metadata Field | Description
+--- | ---
+`start`| start time, in steps or unix timestamp
+`end`| end time, in steps or unix timestamp
+`resolution`| the pixel resolution,by the power of two (1/4, 1/2,... 2, 4, 16), for a scale of 256 x 256 pixels<br /><br />**Note:** TileCubes are typically rendered on tiles of 256 x 256 pixels. It is recommended that you choose a scale that renders perfectly along the borders of the 256 x 256 tile, otherwise there may be issues rendering artifacts. The available pixel resolutions are: `1, 2, 4, 8, 16, 32, 64, 128, 256`
+`data_steps`| number of steps (in integer format)
+`column_type`| "integer" or "date", default "integer"
+`minzoom`| minimum zoom level, optional
+`maxzoom`| max zoom level, optional
+`tiles`| tile array for this set, **required**
+`bounds`| [bounding box](http://wiki.openstreetmap.org/wiki/Bounding_Box) for tileset, optional
+
+#### Example of Metadata Document
+
+{% highlight js %}
```

Comment:
Okay, I am not sure what this is as I simply followed the format of other examples. @matallo , can you advise?